prboom2/src/dsda/console.c

```diff
@@ -254,6 +260,13 @@ static void dsda_ExecuteConsole(void) {
         break;
       }
     }
+
+    char prefix[2];
```

Comment:
In C, variables need to be declared at the top of the scope. Doing it in the middle of a scope is a c++ feature (though some compilers will let it slide).